cylc/flow/cfgspec/workflow.py

```diff
@@ -1731,57 +1732,34 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                 ''')
 
             with Conf('workflow state polling', desc=f'''
-                Configure automatic workflow polling tasks as described in
-                :ref:`WorkflowStatePolling`.
-
-                The items in this section reflect
-                options and defaults of the ``cylc workflow-state`` command,
-                except that the target workflow ID and the
-                ``--task``, ``--cycle``, and ``--status`` options are
-                taken from the graph notation.
+                Deprecated support for automatic workflow state polling tasks
+                as described in :ref:`WorkflowStatePolling`. Note the Cylc 7
+                "user" and "host" config items are not supported.
 
                 .. versionchanged:: 8.0.0
 
                    {REPLACES}``[runtime][<namespace>]suite state polling``.
-            '''):
-                Conf('user', VDR.V_STRING, desc='''
-                    Username of your account on the workflow host.
 
-                    The polling
-                    ``cylc workflow-state`` command will be
-                    run on the remote account.
-                ''')
-                Conf('host', VDR.V_STRING, desc='''
-                    The hostname of the target workflow.
+                .. deprecated:: 8.3.0
 
-                    The polling
-                    ``cylc workflow-state`` command will be run there.
-                ''')
+                Please use the workflow_state xtrigger instead:
+                :py:mod:`cylc.flow.xtriggers.workflow_state`.
```

Comment:
Docs build error (on https://github.com/cylc/cylc-doc/pull/727):
```
Warning, treated as error:
<unknown>:2779:py:mod reference target not found: cylc.flow.xtriggers.workflow_state
```

```suggestion
                Please use the :ref:`workflow_state xtrigger
                <Built-in Workflow State Triggers>` instead:
```

Reply:
Hmm, thought I had fixed this. I must not have pushed the commit from my laptop (late night).